kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/KieServerSynchronization.java

```diff
@@ -92,6 +93,27 @@ public static void waitForKieServerSynchronization(final KieServicesClient clien
         });
     }
 
+    public static void waitForKieServerMessage(final KieServicesClient client, final String containerId, final String expectedMessage) throws Exception {
+        waitForCondition(() -> {
+            ServiceResponse<KieContainerResource> containerResponse = client.getContainerInfo(containerId);
+            if (!containerResponse.getType().equals(ServiceResponse.ResponseType.SUCCESS)) {
+                return false;
+            }
+
+            // Kie Container store only one message
+            List<Message> messagesList = containerResponse.getResult().getMessages();
+            if (messagesList.size() == 1) {
+                for (String message : messagesList.get(0).getMessages()) {
```

Comment:
Just an idea, this cycle can be replaced by "return messagesList.get(0).getMessages().stream().anyMatch(n -> n.equals(expectedMessage));"